# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: widget/gtk/MozContainer.cpp
# Commit: e841ad55e443
# Full Hash: e841ad55e443cbf40d9ca3a975e9ff1a6fad1b45
# Author: stransky <stransky@redhat.com>
# Date: 2023-12-06 05:18:37
# Description:
#   Bug 1868038 [Linux] Clean up GdkWindow at moz_container_unrealize() r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D195524
# ==============================================================================

diff -r 4ed0a5b3a833 -r e841ad55e443 widget/gtk/MozContainer.cpp
--- a/widget/gtk/MozContainer.cpp	Tue Dec 05 20:57:12 2023 +0000
+++ b/widget/gtk/MozContainer.cpp	Tue Dec 05 21:05:50 2023 +0000
@@ -240,6 +240,7 @@
                 (void*)window));
 
   gtk_widget_unregister_window(widget, window);
+  gtk_widget_set_window(widget, nullptr);
   gdk_window_destroy(window);
   gtk_widget_set_realized(widget, false);
 }
